worker/src/main/scala/org/apache/celeborn/service/deploy/worker/Worker.scala

```diff
@@ -717,47 +718,23 @@ private[celeborn] class Worker(
         workerInfo.releaseSlots(shuffleKey)
         val applicationId = Utils.splitShuffleKey(shuffleKey)._1
         if (!workerInfo.getApplicationIdSet.contains(applicationId)) {
-          // When the running applications does not contain the application corresponding to expired shuffle key,
-          // resource consumption source should remove lose application gauges.
-          removeAppResourceConsumption(applicationId)
-          removeAppActiveConnection(applicationId)
```

Comment:
Removal of these methods removeAppResourceConsumption and removeAppActiveConnection is irrelated with this PR. Why should them be removed?

Reply:
@FMX, `removeAppResourceConsumption` has no usage. Meanwhile, `removeAppActiveConnection` is updated to batch removal of application ids.